This switches bootsplash to verbose before reading the image so
that the user can see what is happening - and if nothing is happening
he can see where it is hanging.

   seife

--- linux/kernel/power/swsusp.c-orig	2004-08-30 09:06:40.000000000 +0200
+++ linux/kernel/power/swsusp.c	2004-08-31 10:55:31.000000000 +0200
@@ -1204,6 +1204,14 @@
 
 	if ((error = verify()))
 		return error;
+
+#ifdef CONFIG_BOOTSPLASH
+	{
+		extern int splash_verbose(void);
+		(void)splash_verbose();
+	}
+#endif
+
 	if ((error = read_pagedir()))
 		return error;
 	if ((error = data_read())) {
